sutime/sutime.py

```diff
@@ -90,7 +90,8 @@ def __init__(
         self._sutime = None
         self._lock = threading.Lock()
         module_root = Path(__file__).resolve().parent
-        self.jars = Path(jars) if jars else module_root / 'jars'
+        self.jars = Path(jars) if jars else os.path.join(
```

Comment:
You're mixing `os.path` and `pathlib`.
It should be 
```
Path(jars) if jars else Path(importlib.util.find_spec('sutime').origin).parent / 'jars'
```

Does `Path(__file__).resolve().parent` not resolve correctly to the module root for you?